nixos/modules/services/home-automation/home-assistant.nix

```diff
@@ -388,9 +446,24 @@ in {
           rm -f "${cfg.configDir}/ui-lovelace.yaml"
           ln -s ${lovelaceConfigFile} "${cfg.configDir}/ui-lovelace.yaml"
         '';
+        copyCustomLovelaceModules = if cfg.customLoveLaceModules != [] then ''
+          mkdir -p "${cfg.configDir}/www"
+          ln -sf ${customLovelaceModulesDir} "${cfg.configDir}/www/nixos-lovelace-modules"
+        '' else ''
+          rm "${cfg.configDir}/www/nixos-lovelace-modules"
```

Comment:
Does this need to be `rm -f` or am I misunderstanding how this works?

Reply:
This is a symlink, so it should just go away with a simple `rm`.

Reply:
But will it always exist?

Reply:
Fair point. Thanks, I'll rework the control flow I guess.

Reply:
Forcing deletion, not sure since when.